features/step_definitions/cucumber_rails_steps.rb

```diff
@@ -98,9 +97,12 @@ def fixture(path)
 Given /^I force selenium to run Firefox in headless mode$/ do
   selenium_config = %{
     Capybara.register_driver :selenium do |app|
+      http_client = Selenium::WebDriver::Remote::Http::Default.new
```

Comment:
This isn't directly needed. But to hazard a guess is this extra timeout something that causes intermittent failures>?

Reply:
Read the PR description! :)